yt_dlp/extractor/navernow.py

```diff
@@ -0,0 +1,198 @@
+from .naver import NaverBaseIE
+from ..utils import (
+    ExtractorError,
+    unified_strdate,
+    unified_timestamp,
+    merge_dicts,
+)
+from ..compat import (
+    compat_urllib_parse_urlparse,
+    compat_parse_qs,
+)
+
+
+class NaverNowIE(NaverBaseIE):
+    IE_NAME = 'navernow'
+    _VALID_URL = r'https?://now\.naver\.com/show/(?P<id>[0-9]+)'
+    _PAGE_SIZE = 30
+    _TESTS = [{
+        'url': 'https://now.naver.com/show/4759?shareReplayId=5901#replay=',
+        'info_dict': {
+            'id': '4759-5901',
+            'title': '아이키X노제\r\n💖꽁냥꽁냥💖(1)',
+            'ext': 'mp4',
+            'thumbnail': r're:^https?://.*\.jpg',
+            'timestamp': 1650369600,
+            'upload_date': '20220419',
+            'uploader_id': 'now',
+            'uploader': '',
+            'uploader_url': '',
+            'view_count': int,
+        },
+        'params': {
+            'noplaylist': True,
+            'skip_download': True,
+        }
+    }, {
+        'url': 'https://now.naver.com/show/4759?shareHightlight=1078#highlight=',
+        'info_dict': {
+            'id': '4759-1078',
+            'title': '아이키: 나 리정한테 흔들렸어,,, 질투 폭발하는 노제 여보😾 [아이키의 떰즈업]ㅣ네이버 NOW.',
+            'ext': 'mp4',
+            'thumbnail': r're:^https?://.*\.jpg',
+            'upload_date': '20220504',
+            'timestamp': 1651648042,
+            'uploader_id': 'now',
+            'uploader': '',
+            'uploader_url': '',
+            'view_count': int,
+        },
+        'params': {
+            'noplaylist': True,
+            'skip_download': True,
+        },
+    }, {
+        'url': 'https://now.naver.com/show/4759',
+        'info_dict': {
+            'id': '4759',
+            'title': '아이키의 떰즈업',
+        },
+        'playlist_mincount': 48
+    }, {
+        'url': 'https://now.naver.com/show/4759?shareReplayId=5901#replay',
+        'info_dict': {
+            'id': '4759',
+            'title': '아이키의 떰즈업',
+        },
+        'playlist_mincount': 48,
+        'params': {
+            'noplaylist': False,
+        }
+    }, {
+        'url': 'https://now.naver.com/show/4759?shareHightlight=1078#highlight=',
+        'info_dict': {
+            'id': '4759',
+            'title': '아이키의 떰즈업',
+        },
+        'playlist_mincount': 48,
+        'params': {
+            'noplaylist': False,
+        }
+    }]
+
+    def _call_api(self, path, video_id, query=None, note=None):
+        if note is None:
+            note = 'Downloading JSON metadata'
+        return self._download_json(
+            'https://apis.naver.com/now_web/nowcms-api-xhmac/cms/v1' + path, video_id,
+            note, query=query)
```

Comment:
```suggestion
    def _call_api(self, path, video_id, **kwargs):
        return self._download_json(
            'https://apis.naver.com/now_web/nowcms-api-xhmac/cms/v1/{path}', video_id, **kwargs)
```

Better to save the URL in `self._API_URL` and call `_download_json` directly imho